src/design/optForce/analizeOptForceSol.m

```diff
@@ -0,0 +1,131 @@
+function [maxGrowthRate, minTarget, maxTarget] = analizeOptForceSol(model, targetRxn, solution, relax, tol)
```

Comment:
@tpfau, the `src/design/optForce/analizeOptForceSol.m` file is a new file.

For your information, the MATLAB.devTools recognise when a move of a file is made (e.g. just happened on PR #525).